tests/conftest.py

```diff
@@ -31,7 +31,7 @@ def pytest_configure():
         },
         SITE_ID=1,
         SECRET_KEY='not_very_secret_in_tests',
-        WAGTAIL_SITE_NAME = 'test-site',
+        WAGTAIL_SITE_NAME='test-site',
```

Comment:
Furthermore, add ``tests`` in the ``tox.ini`` package because that folder isn't in the files path. You should also ask Travis to run the ``flake8`` environment otherwise code styles issues are not spotted in others PRs.